docs/examples/quick-start/index.md

```diff
@@ -61,8 +73,20 @@ Make sure all the code is called after the `div` and `leaflet.js` inclusion. Tha
 
 It's worth noting that Leaflet is provider-agnostic, meaning that it doesn't enforce a particular choice of providers for tiles. Also, Leaflet doesn't even contain a single provider-specific line of code, so you're free to use other providers if you need to.
 
-Whenever using anything based on OpenStreetMap, an *attribution* is obligatory as per the [copyright notice](https://www.openstreetmap.org/copyright). Most other tile providers (such as [Mapbox](https://docs.mapbox.com/help/how-mapbox-works/attribution/), [Stamen](http://maps.stamen.com/) or [Thunderforest](https://www.thunderforest.com/terms/)) require an attribution as well. Make sure to give credit where credit is due.
+Whenever using anything based on OpenStreetMap, an attribution is obligatory as per the copyright notice. 
+
+####Map Tiles
+Leaflet does not provide a tile layer, it uses OpenStreetMap. Please adhere to OpenStreetMap's [copyright notice](https://www.openstreetmap.org/copyright) and usage guidelines. Other tile providers include:
+	*[Mapbox](https://docs.mapbox.com/help/how-mapbox-works/attribution/)
+	*[Stamen](http://maps.stamen.com/)
+	*[Thunderforest](https://www.thunderforest.com/terms/)
+	*[Maptiler](https://www.maptiler.com/)
+Please give credit where it is due.
 
+If you wish to use your own map tiles, please refer to the following tile layer format.
+	'https://{s}.somedomain.com/blabla/{z}/{x}/{y}{r}.png'
```

Comment:
It'd look nicer with a [reserved example domain](https://www.iana.org/domains/reserved) such as `example.org` as opposed to `somedomain.com/blabla/`.